### ISSUE TITLE
Outline doesn't refresh on ipynb file updates 

### ISSUE THREAD

**Issue by vladpy8** [microsoft/vscode#190262]
Type: <b>Bug</b>

1. Just create plain .ipynb file 
2. Open outline view on side panel
3. Create few markdown cells with random text/title in .ipynb 
4. Observe no changes in outline

Outline updates in versions of vscode prior to 1.80

VS Code version: Code 1.81.1 (6c3e3dba23e8fadc360aed75ce363ba185c49794, 2023-08-09T22:18:39.991Z)
OS version: Linux x64 6.2.0-26-generic
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1255U (12 x 3910)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|3, 1, 1|
|Memory (System)|38.86GB (34.59GB free)|
|Process Argv|--unity-launch --crash-reporter-id 8f31ea37-c98e-44e7-885b-aa0d3acb36e9|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu-xorg|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu-xorg|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
git-graph|mhu|1.30.0
isort|ms-|2023.10.1
pylint|ms-|2023.6.0
python|ms-|2023.14.0
vscode-pylance|ms-|2023.8.20
jupyter|ms-|2023.7.1002162226
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
even-better-toml|tam|0.19.2
markdown-all-in-one|yzh|3.5.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263cf:30792227
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30803844
2e4cg342:30602488
f6dab269:30613381
a9j8j154:30646983
showlangstatbar:30737416
vsctsb:30748421
03d35959:30757346
57b77579:30736110
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
gsofa:30804715
pythonnosmt12:30797651
pythonidxptcf:30805731
pythonnoceb:30805159
dsvsc013:30795093
dsvsc014:30804076
diffeditorv2:30810526

```

</details>

**Comment by vladpy8** [comment]
Hello, any updates on the issue? It is a bit pesky to not being able to use brilliant Outline View in latest version of VSCode
Is is reproducible so far?

**Comment by clusters9** [comment]
It occurs to me too, and the 'follow the cursor' in the outline also doesn't work in Jupyter notebook.
My platform is macOS Ventura 13.4.1

**Comment by zachmeador** [comment]
I'm also experiencing this issue, and it seems somewhat recent. It seems the only way to get a refreshed outline now is to close/reopen the ipynb, which is obviously a pain because that kills the kernel. Sad to see the this many regressions with the Jupyter notebook functionality. 

VSCode 1.81.1
Windows Server 10.0.17763

**Comment by nicolaeH10** [comment]
> I'm also experiencing this issue, and it seems somewhat recent. It seems the only way to get a refreshed outline now is to close/reopen the ipynb, which is obviously a pain because that kills the kernel. Sad to see the this many regressions with the Jupyter notebook functionality.
> 
> VSCode 1.81.1 Windows Server 10.0.17763

I am experiencing the same issue as well. There's no need to restart vscode though; as a workaround, you can try to double click **Outline** bar (hide/unhide). Surely, this issue needs a proper fix.

version: VSCode 1.81.1
OS: Ubuntu 22.04
kernel: x86_64 Linux 6.2.0-26-generic

**Comment by Yoyokrazy** [comment]
I believe this was actually fixed with https://github.com/microsoft/vscode/pull/191135

The PR wasn't linked to the issue, but this issue is also being closed as a duplicate of #190631

The fix will be in the coming release, and is already in insiders.
